doc/Manuals/ig-ss-xroad_v8_security_server_kubernetes_installation_guide.md

```diff
@@ -0,0 +1,179 @@
+  --set server.extraEnvironmentVars.BAO_PG_CONNECTION_URL="postgres://openbao:\$(BAO_PG_PASSWORD)@db-openbao.ss.svc.cluster.local:5432/openbao"
+```
+
+### 4. Deploy Security Server Databases
+
+```bash
+# Serverconf database
+helm install serverconf-db bitnami/postgresql \
+  --version 18.0.12 \
+  --namespace security-server \
+  --set fullnameOverride=db-serverconf \
+  --set image.repository=bitnamilegacy/postgresql \
+  --set image.tag=16.6.0 \
+  --set auth.database=serverconf \
+  --set auth.username=serverconf \
+  --set auth.password=<serverconf_user_password> \
+  --set auth.enablePostgresUser=true \
+  --set auth.postgresPassword=<postgres_password> \
+  --set primary.resources.requests.memory=64Mi \
+  --set primary.resources.limits.memory=256Mi
+
+# Messagelog database
+helm install messagelog-db bitnami/postgresql \
+  --version 18.0.12 \
+  --namespace security-server \
+  --set fullnameOverride=db-messagelog \
+  --set image.repository=bitnamilegacy/postgresql \
+  --set image.tag=16.6.0 \
+  --set auth.database=messagelog \
+  --set auth.username=messagelog \
+  --set auth.password=<messagelog_user_password> \
+  --set auth.enablePostgresUser=true \
+  --set auth.postgresPassword=<postgres_password> \
+  --set primary.resources.requests.memory=64Mi \
+  --set primary.resources.limits.memory=256Mi
+
+# Operational Monitor database
+helm install opmonitor-db bitnami/postgresql \
+  --version 18.0.12 \
+  --namespace security-server \
+  --set fullnameOverride=db-opmonitor \
+  --set image.repository=bitnamilegacy/postgresql \
+  --set image.tag=16.6.0 \
+  --set auth.database=op-monitor \
+  --set auth.username=opmonitor \
+  --set auth.password=<messagelog_user_password> \
+  --set auth.enablePostgresUser=true \
+  --set auth.postgresPassword=<postgres_password> \
+  --set primary.resources.requests.memory=64Mi \
+  --set primary.resources.limits.memory=256Mi
+```
+
+### 5. Deploy Security Server
+
+```bash
+helm install security-server xroad/security-server \
+  --version 8.0.0-beta1 \
+  --namespace security-server \
+  --set init.serverconf.dbUsername=serverconf \
+  --set init.serverconf.proxyUiSuperuser=<proxy_ui_superuser> \
+  --set init.serverconf.proxyUiSuperuserPassword=<proxy_ui_superuser_password> \
+  --set init.messagelog.dbUsername=messagelog \
+  --set init.opmonitor.dbUsername=opmonitor \
+  --set services.proxy.env.XROAD_PROXY_ADDON_OP_MONITOR_ENABLED=true \
+  --set services.op-monitor.enabled=true \
+  --set services.backup-manager.env.SERVERCONF_INITIALIZED_WITH_PROXY_UI_SUPERUSER=true \
+  --set services.backup-manager.env.PROXY_UI_SUPERUSER=xrd
+```
+
+>**Note:**
+>* `proxyUiSuperuser` is the initial user (with all the privileges) which can be used to login into Proxy UI admin webapp.
+>* `init.serverconf.proxyUiSuperuserPassword` must be provided in argon2 hash format. For example argon2 hash of `secret` is: `$argon2id$v=19$m=16384,t=2,p=1$YXF3YXN6eHh6c2F3cQ$+llp8EbxlqZaF2uO/BLoFLwfqxe1Yn6BvC/DOegq6A0`.
```

Comment:
It would be good to provide a link to a tool or a command line command that can be used to generate the hash in the required form.